src/shared/api/dto.ts

```diff
@@ -128,7 +128,7 @@ declare global {
     /**
      * List of serialized {@link @iroha2/data-model#VersionedValidTransaction}
      */
-    transactions: string[];
+    transactions: TransactionDto[];
```

Comment:
..why?

Look at the `BlockDTO` at the backend:

https://github.com/soramitsu/iroha2-block-explorer-backend/blob/57da68535e2043e0694ae4049ffac83bae2565d1/src/web/blocks.rs#L64

It has `transactions: Vec<SerScaleHex<TransactionValue>>`. Thus, in JSON it will be `string[]`.